projects/apardpfwd/src/examples/apard_communication_example/apard_communication_example.c

```diff
@@ -0,0 +1,151 @@
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+ * EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*******************************************************************************/
+
+#include <stdio.h>
+#include "common_data.h"
+
+#include "lwip_socket.h"
+#include "tcp_socket.h"
+#include "no_os_error.h"
+#include "no_os_gpio.h"
+#include "adin1110.h"
+#include "network_interface.h"
+
+/***************************************************************************//**
+ * @brief Configure the output port of the AD-APARDPFDW-SL then open a TCP socket
+ *		  to communicate with the AD-APARD32690-SL.
+ * @return ret - Result of the example execution.
+*******************************************************************************/
+
+int apard_communication_example_main()
+{
+	struct no_os_uart_desc *uart_desc;
+
+	uint32_t device_id;
+	int ret;
+
+	ret = no_os_uart_init(&uart_desc, &uart_ip);
+	if (ret)
+		return ret;
+
+	no_os_uart_stdio(uart_desc);
+
+	ret = spi_cfg_0(adin1110_spi_cfg_0);
+	if (ret) {
+		printf("Error setting the reset GPIO direction (%d)\n", ret);
+		return ret;
+	}
+
+	ret = port2_cfg(port2_cfg_0, NO_OS_GPIO_HIGH);
+	if (ret) {
+		printf("Error setting the reset GPIO direction (%d)\n", ret);
+		return ret;
+	}
+
+	printf("AD-APARDPFWD APARD COMMUNICATION EXAMPLE.\n");
+
+	struct tcp_socket_desc *server_socket;
+	struct tcp_socket_desc *client_socket;
+	struct lwip_network_desc *lwip_desc;
+	struct tcp_socket_init_param tcp_ip = {
+		.max_buff_size = 0
+	};
+
+	memcpy(lwip_ip.hwaddr, adin1110_ip.mac_address, NETIF_MAX_HWADDR_LEN);
+
+	ret = no_os_lwip_init(&lwip_desc, &lwip_ip);
+	if (ret)
+		return ret;
+
+	printf("\rMAC address from mac_desc: %02X:%02X:%02X:%02X:%02X:%02X\n",
+	       ((struct adin1110_desc*) lwip_desc->mac_desc)->mac_address[0],
+	       ((struct adin1110_desc*) lwip_desc->mac_desc)->mac_address[1],
+	       ((struct adin1110_desc*) lwip_desc->mac_desc)->mac_address[2],
+	       ((struct adin1110_desc*) lwip_desc->mac_desc)->mac_address[3],
+	       ((struct adin1110_desc*) lwip_desc->mac_desc)->mac_address[4],
+	       ((struct adin1110_desc*) lwip_desc->mac_desc)->mac_address[5]);
+
+	ret = adin1110_reg_read((struct adin1110_desc*) lwip_desc->mac_desc, 0x1,
+				&device_id);
+	if (ret) {
+		printf("Error reading the ADIN1110's device id (%d)\n", ret);
+		goto out;
+	}
+
+	printf("\rGot device id 0x%X\n\r", device_id);
+
+	tcp_ip.net = &lwip_desc->no_os_net;
+
+	ret = socket_init(&server_socket, &tcp_ip);
+	if (ret)
+		return ret;
+
+	ret = socket_bind(server_socket, 10000);
+	if (ret)
+		return ret;
+
+	ret = socket_listen(server_socket, MAX_BACKLOG);
+	if (ret)
+		return ret;
+
+	bool connected = false;
+	uint8_t read_byte;
+
+	while (1) {
+		ret = socket_accept(server_socket, &client_socket);
+		if (ret && ret != -EAGAIN)
+			return ret;
+
+		if (!ret) {
```

Comment:
no brackets if guarding only one statement. check all occurences throughout the code.